README.md

```diff
@@ -1,12 +1,58 @@
-# MeetupTalks
-
-Python Pune monthly talks
-
-# 2019
-## [January](./2019/January/README.md)
-## [February](./2019/February/README.md)
-## [March](./2019/March/README.md)
-## [April](./2019/April/README.md)
-## [May](./2019/May/README.md)
-## [June](./2019/June/README.md)
-## [July](./2019/July/README.md)
+# PythonPune Meetup Talks
+
+Welcome to meetup repository of PythonPune. This repository contains
+details of monthly events conducted as part of this group.
+
+* [Submit a talk](#submit-a-talkworkshop)
+* [Monthly newsletter](#monthly-newsletter)
+* [Summary of past events](#summary-of-past-events)
+* [Code of Conduct](#code-of-conduct)
+* [License](#license)
+
+## Submit a talk/workshop
+If you want to submit a talk for upcoming events, then you can create
+a new issue on this repository. Just fill in the requested details and
```

Comment:
How about we hyper-link the text `create a new issue` with URL https://github.com/pythonpune/meetup-talks/issues/new/choose?

Reply:
Right, I forgot to do that.